Cargo.toml

```diff
@@ -73,7 +73,6 @@ bip39 = { version = "2.0.0", features = ["serde"] }
 time-util = { version = "0.3.4", features = ["chrono"] }
 assert-json-diff = "2.0.2"
 url = { version = "2.5.0", features = ["serde"] }
-num-format = "0.4.4"
```

Comment:
wont use identifier based, will let iOS  / Android specify decimal separator and grouping separator.